pkg/term/windows/ansi_reader.go

```diff
@@ -10,8 +10,8 @@ import (
 	"strings"
 	"unsafe"
 
-	. "github.com/Azure/go-ansiterm"
-	. "github.com/Azure/go-ansiterm/winterm"
+	ansiterm "github.com/Azure/go-ansiterm"
```

Comment:
Yeah, I don't think the go lexer can handle - in identifiers. Probably we shouldn't have called it go-ansiterm, but it's too late to fix that. This alias is perfectly reasonable.